packages/common/src/ui/components/menus/DropdownMenu/DropdownMenu.tsx

```diff
@@ -80,25 +80,27 @@ export const DropdownMenu: FC<DropdownMenuProps> = ({
   children,
   disabled = false,
 }) => {
-  return (
-    <FormControl size="small">
-      <InputLabel
-        shrink={false}
-        sx={{ color: '#8f90a6', '&.Mui-focused': { color: '#8f90a6' } }}
-        id={`action-drop-down-label-${label}`}
-      >
-        {label}
-      </InputLabel>
-      <StyledSelect
-        disabled={disabled}
-        value=""
-        size="small"
-        labelId={`action-drop-down-label-${label}`}
-        variant="outlined"
-        IconComponent={ChevronDownIcon}
-      >
-        {children}
-      </StyledSelect>
-    </FormControl>
-  );
+  if (Array.isArray(children))
+    return (
+      <FormControl size="small">
+        <InputLabel
+          shrink={false}
+          sx={{ color: '#8f90a6', '&.Mui-focused': { color: '#8f90a6' } }}
+          id={`action-drop-down-label-${label}`}
+        >
+          {label}
+        </InputLabel>
+        <StyledSelect
+          disabled={disabled}
+          value=""
+          size="small"
+          labelId={`action-drop-down-label-${label}`}
+          variant="outlined"
+          IconComponent={ChevronDownIcon}
+        >
+          {children}
+        </StyledSelect>
+      </FormControl>
+    );
+  else return <div>{children}</div>;
```

Comment:
if we do stick with the button - it was looking a little lost, and I wondered about this:

```
const StyledContainer = styled('div')(({ theme }) => ({
  borderWidth: 1,
  borderStyle: 'solid',
  borderColor: theme.palette.gray.light,
  borderRadius: 5,
  '& > li': { height: '100%' },
}));
```

and 
```
  else return <StyledContainer>{children}</StyledContainer>;
```

giving

<img width="418" alt="Screen Shot 2022-03-25 at 3 21 11 PM" src="https://user-images.githubusercontent.com/9192912/160041881-7601abed-3447-45a2-b97f-20cb26d57b61.png">

Reply:
Yeah, much better. Have made reference to this on [new issue](#1115)